src/js/pages/system/UnitsHealthNodeDetail.js

```diff
@@ -33,12 +43,22 @@ class UnitsHealthNodeDetail extends mixin(StoreMixin) {
     UnitHealthStore.fetchUnitNode(unitID, unitNodeID);
   }
 
-  getErrorNotice() {
-    return (
-      <div className="container container-pod">
-        <RequestErrorMsg />
-      </div>
-    );
+  onUnitHealthStoreUnitSuccess() {
+    let isLoading = this.state.isLoading & ~FLAGS.UNIT_LOADED;
```

Comment:
Not gonna lie, the bitwise stuff in here is confusing to me. But I should understand it better anyway so I'm fine with keeping it.